### ISSUE TITLE
Automatic package versioning

### ISSUE THREAD

**Issue by kilemensi** [CodeForAfrica/ui#229]
Our current release process seem to be manual and random. As part of release discussion, we lean towards two approaches:

 - Release a new version on `every` merge to main, and
 - Release whenever there is something **significant** to release.

Having read upon this a bit further, our true crime is the manual and random part. Whether its on every merge on when there is something **significant** to release, doesn't really matter much in the end.

Having read on [Changesets](https://github.com/changesets/changesets) again (did a bit of it at the time of setting up this repo), it seems like it can help us automate and bring predictability to our releases. I'd urge everyone of us to at least read the [intro](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md) and [automating changesets](https://github.com/changesets/changesets/blob/main/docs/automating-changesets.md) if they don't have time to read through the full docs.

In a nutshell, we'll add a custom script to bump version of Dockerfile after changesets version has bumped package.json. Similar to [`changesets/action`](https://github.com/changesets/action/blob/main/scripts/bump.js) updates changelog. Changesets will then create a PR with these changes.

Let me know what you guys think @esirK, @kelvinkipruto, @thepsalmist

**Comment by kilemensi** [comment]
Resolved by #232
